lib/nylas/account.rb

```diff
@@ -0,0 +1,31 @@
+module Nylas
+  # Representation of the accounts for Account management purposes.
+  # @see https://docs.nylas.com/reference#account-management
+  class Account
+    include Model
+    self.searchable = false
+    self.read_only = true
+
+    attribute :id, :string
+    attribute :account_id, :string
+    attribute :billing_state, :string
+    attribute :sync_state, :string
+
+    attribute :email, :string
+    attribute :trial, :boolean
+
+    def activate
```

Comment:
Hmm, I think we should keep the same terminology as our existing API – why not rename these methods `upgrade!` and `downgrade!`?